portfolio/src/main/webapp/script.js

```diff
@@ -34,4 +34,22 @@ function pickRandomQuote() {
 function divMessage() {
     const quoteToDisplay = pickRandomQuote();
     document.getElementById('quote-displayer').innerText = quoteToDisplay;
+}
+
+function fetchMyName() {
+    console.log('Fetching your name...');
+    const responsePromise = fetch('/data');
+    responsePromise.then(handleResponse);
```

Comment:
FYI, you can use anonymous functions here and keep the whole name fetching process in a single function.